cypress/support/users.js

```diff
@@ -0,0 +1,10 @@
+import { Button, Select, TextField } from "../../interactors";
+
+Cypress.Commands.add('createUser', (userLastName, patronGroup, email) => {
+  cy.do([
+    TextField('Last name\n*').fillIn(userLastName),
```

Comment:
id/attr is more preferable

Reply:
You can also a matcher if you don't want to include the `\n*`:

```js
import { including } from "../../interactors"

// .....
cy.do([
  TextField(including('Last name')).fillIn(userLastName),
  //....
]);